src/utils/v2/fundingCycle.ts

```diff
@@ -147,3 +155,68 @@ export const decodeV2FundingCycleMetadata = (
 
   return metadata
 }
+
+/**
+ * Mark various funding cycle properties as "unsafe",
+ * based on a subjective interpretation.
+ *
+ * If a value in the returned object is true, it is potentially unsafe.
+ */
+export const getUnsafeV2FundingCycleProperties = (
```

Comment:
Bit of duplication in this file, but I'd rather keep it this way for now. We don't know how much v2 will diverge from V1.

Reply:
Yeah good call I reckon